loom/common/core/performance.cpp

```diff
@@ -606,6 +618,8 @@ static void LoomProfilerEntryDumpRecurse(LoomProfilerEntry *data, char *buffer,
 
 void LoomProfiler::dump()
 {
+    LOOM_PROFILE_ZERO_CHECK()
```

Comment:
It just calls `hashZeroCheck` from anywhere and follows the style of `LOOM_PROFILE_START/END`.